src/services/tx/txMonitor.ts

```diff
@@ -33,3 +36,52 @@ export const waitForTx = async (provider: JsonRpcProvider, txId: string, txHash:
     })
   }
 }
+
+export const waitForRelayedTx = (taskId: string, txId: string): void => {
```

Comment:
I think we should add a limit on this like we do in `waitForTx`. If it's taking longer than usual, we remove the status. Wdyt?

Reply:
I checked the Gelato docs and did not find much information about expected times for exectution. 
The only comment I found is about the `WaitingForConfirmation` state:
> the relay request has been received by Gelato Relay and it is waiting to be confirmed as a task in the relay task queue. This state usually lasts only a few seconds.

So I guess for that one we could have a ~30s timeout.  But for execution the timeout should rather be too big than too small.

Reply:
_Question:_ as you can't cancel the relaying transaction, should we display something along the lines of the `waitForTx` timeout msg?

```
throw new Error(
  `Transaction not relayed in ${TIMEOUT_MINUTES} minutes. Be aware that it might still be relayed.`,
)
```

Reply:
I've added a timeout of 3 min and added a test for it